app/assets/stylesheets/redde-admin/redactor/index.scss

```diff
@@ -129,14 +132,14 @@ body .redactor_toolbar li a.redactor_btn_html span				  { background-position: 0
 body .redactor_toolbar li a.redactor_btn_styles span			  { background-position: -25px; }
 body .redactor_toolbar li a.redactor_btn_bold span				  { background-position: -50px; }
 body .redactor_toolbar li a.redactor_btn_italic span			  { background-position: -75px; }
-body .redactor_toolbar li a.redactor_btn_deleted span		 	  { background-position: -550px; }		
-body .redactor_toolbar li a.redactor_btn_insertunorderedlist span { background-position: -100px; }		
+body .redactor_toolbar li a.redactor_btn_deleted span		 	  { background-position: -550px; }
+body .redactor_toolbar li a.redactor_btn_insertunorderedlist span { background-position: -100px; }
 body .redactor_toolbar li a.redactor_btn_insertorderedlist span   { background-position: -125px; }
 body .redactor_toolbar li a.redactor_btn_outdent span	 		  { background-position: -150px; }
 body .redactor_toolbar li a.redactor_btn_indent span		 	  { background-position: -175px; }
 body .redactor_toolbar li a.redactor_btn_image span		 		  { background-position: -200px; }
 body .redactor_toolbar li a.redactor_btn_video span		 		  { background-position: -225px; }
-body .redactor_toolbar li a.redactor_btn_file span		 		  { background-position: -250px; }	
+body .redactor_toolbar li a.redactor_btn_file span		 		  { background-position: -250px; }
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 5<br>Selector `redactor_toolbar` should be written in lowercase with hyphens<br>Selector `redactor_btn_file` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space